other/boards.yaml

```diff
@@ -41,8 +41,8 @@ chandalar:
 lifcl-40:
   family: nexus
   device: LIFCL-40-9BG400CES
```

Comment:
@acomodi , do you think `LIFCL-40-9BG400CES` should be broken into device and package?   The "9" is speed grade, and "ES" is engineering sample, so the last field is not strictly about the package, but for e.g. Xilinx parts the "package" is not strictly about the package either.

I'm fine either way.

Reply:
I don't know how this matches to the usage in `fpga-tool-perf` but I would say that package is about how the silicon is broken out to real pins. You can have multiple different devices in the same package and multiple different packages for the same device.

Reply:
Yeah, this pretty much is the correspondence in `fpga-tool-perf`.

- family -> architecture type in terms of structure (e.g. UltraScale+, Series7, nexus, etc.)
- device -> number/disposition of LUTs, BRAMs, DSPs, etc. (a50t, a100t, etc.)
- package -> usable IO pins

A combination of the three is a `part`,